analytics/pyproject.toml

```diff
@@ -11,8 +11,6 @@ analytics = "analytics.cli:app"
 
 [tool.poetry.dependencies]
 dynaconf = "^3.2.4"
-kaleido = "0.2.1"
-notebook = "^7.0.0"                              # Goal is to replace this with another method of presenting charts
```

Comment:
These two dependencies were never imported directly, so as far as I can tell they don't need to be here.